js-arrays/js-arrays.js

```diff
@@ -0,0 +1,53 @@
+// 1) Create an array of colours
+let clrs = ["gold", "purple", "silver", "jet"];
+
+// 2) Loop through the array using the for loop and log the colour to the console
+for (let i = 0; i < clrs.length; i++) console.log(clrs[i]);
+
+// 3) Loop through the array using the forEach loop and log the colour to the console
+clrs.forEach((c) => console.log(c));
+
+// 4) Write a function that will rotate an array to the right by a certain number of "steps"
+const rotate = (arr, num) => {
+    if (num > arr.length) {
+        throw "num cannot be greater than array length";
```

Comment:
U sustini bi bilo dobro da funkcija moze da podrzi i rotiranje sa bilo kakvim brojem koraka pa cak i u situacijama kada treba da se napravi 2 ili vise krugova rotacije. Na primer ako imas niz od 4 elementa, rotacija je ista za 1, 5, 9 koraka itd. 
Pretpostavljam da si uveo to pravilo da broj koraka ne sme biti veci od duzine niza zbog nacina na koji si radio.
Da bi postigao univerzalno resenje, mogao si koristiti kombinaciju `unshift` i `pop` funkcija koje su deo `Array` objekta. Vise o njima, a i suprotnim funkcijama (`shift` i `push`) mozes procitati [ovde](https://codesource.io/array-pop-push-shift-and-unshift-methods-in-javascript/).